tests/Resources/ti.geolocation.test.js

```diff
@@ -244,7 +246,52 @@ describe.windowsBroken('Titanium.Geolocation', function () {
 		should(Ti.Geolocation.trackSignificantLocationChange).be.true();
 	});
 
+	it.ios('.ACCURACY_REDUCED', function () {
+		if (OS_VERSION_MAJOR >= 14) {
```

Comment:
So we define these special filters like `it.ios`, where it skips on other platforms. It may make sense to expand to add ones like `it.ios14Plus` because technically we'd want to skip on older versions, not have a no-op test that always passes. (Alternatively, once we upgrade from ti-mocha to the real mocha package, I think you can explicitly call `this.skip()` inside a test to mark it at runtime)